test/e2e/canary_test.go

```diff
@@ -407,7 +407,7 @@ spec:
 		When().
 		PromoteRollout().
 		WaitForRolloutStatus("Degraded").
-		WaitForRolloutStatus("Healthy")
+		WaitForRolloutStatus("Healthy", time.Second*90)
```

Comment:
Is there a reason for the 90 seconds?

Reply:
This is just to avert the flaky e2e test.